app/schema.py

```diff
@@ -55,9 +84,88 @@ def mutate(root, info, contact, password):
         return LoginUser(logged_in=True, token=make_hash(user))
 
 
+class SendFriendRequest(graphene.Mutation):
+    class Arguments:
+        contact_receiver = graphene.String()
+    ok = graphene.Boolean()
+    user_receiver = graphene.Field(UserType)
+    friend_request = graphene.Field(FriendRequestType)
+
```

Comment:
Have a message field, that gives info about the status of the request, like if a request is successful or not

Reply:
Okay

Reply:
okay